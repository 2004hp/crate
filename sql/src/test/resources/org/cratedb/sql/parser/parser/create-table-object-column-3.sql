CREATE TABLE t3 (col1 object(DYNAMIC) AS (col11 object, col12 integer, col13 object(STRICT) AS (col131 string)))