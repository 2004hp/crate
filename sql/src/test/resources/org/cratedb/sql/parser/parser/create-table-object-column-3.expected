org.cratedb.sql.parser.parser.CreateTableNode@71f29905
name: t3
statementType: CREATE TABLE
lockGranularity: R
existenceCheck: NO_CONDITION
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@6e1c51b4

	[0]:	
	org.cratedb.sql.parser.parser.ObjectColumnDefinitionNode@53838e6
	type: OBJECT
	name: col1
	elementType: null
	objectType: DYNAMIC
	subColumns: 		
		org.cratedb.sql.parser.parser.TableElementList@2e1f86b7

		[0]:		
		org.cratedb.sql.parser.parser.ObjectColumnDefinitionNode@17c57009
		type: OBJECT
		name: col11
		elementType: null
		objectType: DYNAMIC
		subColumns: 			
			org.cratedb.sql.parser.parser.TableElementList@233e6b10

		[1]:		
		org.cratedb.sql.parser.parser.ColumnDefinitionNode@56e6f983
		type: INTEGER
		name: col12
		elementType: null
		[2]:		
		org.cratedb.sql.parser.parser.ObjectColumnDefinitionNode@56073407
		type: OBJECT
		name: col13
		elementType: null
		objectType: STRICT
		subColumns: 			
			org.cratedb.sql.parser.parser.TableElementList@72e8e3b0

			[0]:			
			org.cratedb.sql.parser.parser.ColumnDefinitionNode@17bc72c0
			type: string
			name: col131
			elementType: null
